README.md

```diff
@@ -96,6 +96,12 @@ Example ZLint CLI usage:
 
 	echo "Lint mycert.pem with all of the lints except for ETSI ESI sourced lints"
 	zlint -excludeSources=ETSI_ESI mycert.pem
+    
+    echo "Receive a copy of the full (default) configuration for all configurable lints"
+    zlint -exampleConfing
```

Comment:
So the idea here was there is likely only going to be a small subset of lints that are actually configurable (of course, starting with 0). This makes the following questions difficult to answer - `Which lints are configurable? Is there documentation for these fields? What are their default values? If I am lint author, how do I convey the information to the user?`.

To date, the project has done a pretty good job of making it such that all a new lint has to do is add one new file, one new test file, and any necessary PEM data for tests. That is, everything is isolated making new lints rather easy for people unfamiliar to the codebase to contribute. However, if we naively asked that any new lint configurations kindly update the documentation on the current configuration, then that opens up a world of bookkeeping that can be quite lossy and error prone.

So what I've done here is make the following contract with lint authors...

> The code is going to require that you give me a default anyways, and our TOML framework lets you annotate your struct with comments, so whatever you give me as being your default configuration I will serialize into TOML and show to the end user on your behalf (comments and all).

In this way, new lints don't have to update markdown anywhere or otherwise leave the safety of their one file...

```go
type MyConfig struct {
   TheBestSongInTheWorld string `coment:"Totally your opinion"`
}

func (m (*MyLinT) Configure() interface{} {
    return &MyConfig{TheBestSongInTheWorld: "But I think it's Triangle Man by They Might be Giants"}
}
```

....and automagically that shows up in this "example" default configuration.

```toml
[MyLint]
# Totally your opinion
TheBestSongInTheWorld = "But I think it's Triangle Man by They Might be Giants"
```

Reply:
```suggestion
    zlint -exampleConfig
```